waiter/docs/parameters.md

```diff
@@ -42,6 +42,7 @@ Additional (optional) parameters that can be set:
 |`X-Waiter-Metadata-*`|n/a|any string|A metadata field. Metadata fields allow you to store arbitrary strings along with your service description.|Allows additional information to be stored with a service to help identify the service.|
 |`X-Waiter-Metric-Group`|"other"|[A-Za-z0-9-_]+|Metric groups allow services to be grouped together for the purpose of metric collection.|If you have `:statsd` enabled in your [config file](../config-full.edn), then specify a metric group. If you have multiple services, choose the same metric group if you'd like combined metrics, otherwise choose independent metric groups.|
 |`X-Waiter-Min-Instances`|1|1 or 2|The minimum number of instances that Waiter should keep running at all times.|Go with the default.|
+|`X-Waiter-Namespace`|empty|user-name|The namespace in which to create back-end scheduler objects. Currently only supported on the Kubernetes scheduler, and must match the run-as-user when specified.|Leave this field blank unless you need direct access to resources Waiter creates on your behalf.|
```

Comment:
I copied the default value for X-Waiter-Scheduler. Since I removed the third alternative (fallback to run-as-user) as discussed in another feedback thread, I think this is accurate now.